test/Microsoft.Identity.Abstractions.Tests/AquireTokenOptionsTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Xunit;
+
+namespace Microsoft.Identity.Abstractions.Tests
+{
+    public class AquireTokenOptionsTests
+    {
+        [Fact]
+        public void ManagedIdentity()
+        {
+            // Client credential from Managed Identity
+            // -------------------------------------------------
+            // https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/overview
+            /*
+            // <managedidentity_json>
+            {
+                "AquireTokenOptions": {
+                    "ManagedIdentity": {
+                        "ClientId": "12345"
+                        "ObjectId": "67890"
+                        "HostResourceId": "https://management.azure.com/"
+                    }
+                }
+            }
+            // </managedidentity_json>
+            */
+
+            // <managedidentity_csharp>
+            ManagedIdentityOptions managedIdentityOptions = new ManagedIdentityOptions
+            {
+                ClientId = "12345",
+                ObjectId = "67890",
+                HostResourceId = "https://management.azure.com/"
+            };
+
+            AcquireTokenOptions acquireTokenOptions = new AcquireTokenOptions
+            {
+                ManagedIdentity = managedIdentityOptions
+            };
+            // </managedidentity_csharp>
+
```

Comment:
Could we also have a sample for system-assigned managed identity?

Reply:
Absolutely